news/2023/2023-01-19-gmt-apps-now-open.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Global Moderation Team applications now open!
+date: 2023-01-19 12:00:00 +0000
+---
+
+Been an active community member for a while? Felt an inner urge to chase after some osu! troublemakers? Now it's your time to shine! The Global Moderation Team is seeking new members.
+
+![](/wiki/shared/news/banners/gmt-generic.jpg)
+
+It's been over 2 years since the [last GMT application cycle](https://osu.ppy.sh/home/news/2020-10-04-global-moderation-team-applications-open), and while some things have changed, one thing remains the same — we may be looking for you!
+
+The [Global Moderation Team](https://osu.ppy.sh/groups/4) is responsible for keeping the community in a good condition, keeping it a welcoming place for everyone. Our green-coloured peers of the GMT not only help moderate the in-game chat and forums, but also engage with the mapping community among others. The members of the team can also give you assistance with some of your technical issues. The inherent goal for is to reach closer and closer to the living community.
```

Comment:
> The inherent goal for is to reach closer and closer to the living community.

what?

Reply:
Did you mean to write "us" instead of "is"?

Reply:
ah yes definitely